calicoctl/calico_ctl/profile.py

```diff
@@ -203,6 +218,8 @@ def profile(arguments):
                 protocol=protocol,
                 icmp_type=arguments.get("<ICMPTYPE>"),
                 icmp_code=arguments.get("<ICMPCODE>"),
+                icmpv6_type=arguments.get("<ICMPV6TYPE>"),
+                icmpv6_code=arguments.get("<ICMPV6CODE>"),
```

Comment:
As mentioned in the libcalico change.  I think this should use icmp_type and icmp_code for both versions